elements/lisk-p2p/test/unit/peer_pool.ts

```diff
@@ -160,6 +161,79 @@ describe('peerPool', () => {
 		});
 	});
 
+	describe('#request', () => {
+		let caughtError: Error;
+		beforeEach(async () => {
+			// @ts-ignore
+			peerPool['_peerSelectForRequest'] = sandbox
```

Comment:
why do we need this `ts-ignore` here?

Reply:
My TS compiler complains if I don't. Maybe it doesn't allow assigning to private properties?